pkg/metastore/kv.go

```diff
@@ -57,61 +122,6 @@ func LocationIDFromKey(key string) string {
 	return key[len(locationsKeyPrefix):]
 }
 
-// MakeLocationID returns a key for the location that uniquely identifies the
-// location. Locations are uniquely identified by their mapping ID and their
-// address and whether the address is folded. If a location address is 0, then
-// the lines are expected to be non empty and to be already resolved as they
-// cannot be asynchronously symbolized. The lines are then taken into the
-// location key.
-func MakeLocationID(l *pb.Location) string {
-	hash := sha512.New512_256()
-
-	hash.Write([]byte(l.MappingId))
-
-	//nolint:errcheck // ignore error as writing to the hash will cannot error
-	binary.Write(hash, binary.BigEndian, l.Address)
-	if l.IsFolded {
```

Comment:
Note, I have dropped this logic because `binary.Write(hash, binary.BigEndian, 1)` and `binary.Write(hash, binary.BigEndian, 0)` produce an empty slice https://go.dev/play/p/4vl0F7x7o5n. Please let me know if that's a bug.

```go
package main

import (
	"bytes"
	"encoding/binary"
	"fmt"
)

func main() {
	b := &bytes.Buffer{}
	binary.Write(b, binary.BigEndian, 1)
	binary.Write(b, binary.BigEndian, 0)

	fmt.Println(b.Bytes())
	// Output:
	// []
}
```